app/section/helpers/createQuestionsContextForOptions.test.js

```diff
@@ -82,6 +82,51 @@ describe('when the question type is of type checkbox-options or radiobutton-opti
     expect(context).toEqual(expectedContext);
   });
 
+  it('should create a context for combobox-options type question with existing data populated', () => {
+    const expectedContext = {
+      questionContext: {
+        id: 'some-question-id',
+        type: 'combobox-options',
+        options: [
+          {
+            text: 'option 1',
+            value: 'option 1',
+            selected: true,
+          },
+          {
+            text: 'option 2',
+            value: 'option 2',
+          },
+          {
+            text: 'option 3',
+            value: 'option 3',
+            selected: true,
```

Comment:
Should there only be one item selected here?

Reply:
I think as many can be selected as you want?